src/utils/hooks/useGetReviewStructureForSection/helpers.tsx

```diff
@@ -87,13 +88,20 @@ const generateReviewStructure: GenerateReviewStructure = ({
   // review info comes from reviewAssignment that's passed to this hook
   newStructure.thisReview = review
 
+  // when changes requested by consolidator (included in thisReviewPreviousResponse OR thisReviewLatestResponse)
+  const isChangeResquest = addChangeRequestForReviewer(newStructure)
+
   newStructure = addIsPendingReview(newStructure, level)
   newStructure = addIsActiveReviewResponse(newStructure)
 
+  if (isChangeResquest) {
```

Comment:
tbh, I preferred previous way tbh, it keeps everything in one place and it keeps it consistent with how progress is calculated in application

Reply:
dont' need to change though

Reply:
Ok. I can change this back. No problem